src/Symfony/Component/HttpKernel/HttpKernel.php

```diff
@@ -250,6 +250,13 @@ private function handleException(\Exception $e, Request $request, int $type): Re
         }
     }
 
+    /*
+     * Returns a more explicit  definition of the supplied variable 
```

Comment:
`Returns an exported string for the specified variable.`

Reply:
actually: `Returns a human-readable string for the specified variable.`

Reply:
Hi @ro0NL , Thank you for reviewing these updates. I've committed a further update to this Pull Request taking your comments into account. I've also updated the opening comment to reference the 3.4 version. 

Please can you review these updates whenever you have a chance to do so?